### PR TITLE
[#227] Only hiring in CHI, NY, DC, SF

### PR DESCRIPTION
**Pull request by hestherchang**
Added a message to state that we're only hiring in Chicago, NYC, DC, SF to set expecations

### CODE REVIEW

File: _positions/product-lead.md [hunk 1]
```diff
@@ -8,6 +8,9 @@ active: true
 ###Role Summary
 An 18F Product Lead is responsible for managing the development, delivery, and continuous improvement of products while establishing and managing our relationship with our partners (clients). As a product lead, you’ll work with the 18F intake team to evaluate potential work and communicate to potential clients, partner with the finance and contracts team to close deals, and work with cross-functional teams to deliver the right product in the right way.
 
+At this time, the Product team is only looking for new members who are able to join the team at an office - Chicago, NYC, DC, San Francisco.
```
Comment (fureigh) [inline _positions/product-lead.md]: A few questions and comments:
- Please change the hyphen to an em-dash (on a Mac, you can type that by pressing `shift-option-hyphen`).
- Let's change `DC` to `D.C.`, per the style guide.
- Do we still technically have a NYC office? If so, let's update https://pages.18f.gov/joining-18f/who-we-are-hiring/ accordingly (in a separate pull request).
- We could parallel the "work out of our offices" phrasing from that page, something like "At this time, we're only hiring Product Leads who are able to work out of our offices in [city list] or [final city]." How does that sound to you?

**Comment by fureigh** [comment]
@hestherchang Thanks for this! I added some questions and comments. You can continue the conversation in that line note or add additional commits to this pull request and assign it back to me when it's ready for review again.

File: _positions/product-lead.md (hunk 1 above)
Comment (hestherchang) [inline _positions/product-lead.md]: - Ok - will change hyphen to an em-dash 
- Ok
- checking on this now
- Ok sounds good

**Comment by fureigh** [comment]
@hestherchang, I made a few other copy changes — if they look good to you, please update the Lever text accordingly. Soon I'll [remove the position descriptions from the codebase](https://github.com/18F/joining-18f/issues/232) because they're now hosted on Lever, but I'll merge this in the meantime.

**Pull request merged**